tests/Functional/ConfigurationTest.php

```diff
@@ -126,22 +128,23 @@ public function testChangeComponentLibrary(): void {
       $this->drupalGet('<front>');
 
       // Assert that we load the EC component library by default.
-      $this->assertLinkContainsHref('/oe_theme/dist/ec/styles/ecl-ec-preset-website.css');
-      $this->assertLinkContainsHref('/oe_theme/dist/ec/styles/ecl-ec-preset-website-print.css');
+      $this->assertLinkContainsHref('/oe_theme/dist/ec/styles/ecl-ec.css');
+      $this->assertLinkContainsHref('/oe_theme/dist/ec/styles/ecl-ec-print.css');
       $this->assertLinkContainsHref('/oe_theme/css/style-ec.css');
 
-      $this->assertScriptContainsSrc('/oe_theme/dist/ec/scripts/ecl-ec-preset-website.js');
+      $this->assertScriptContainsSrc('/oe_theme/dist/ec/scripts/ecl-ec.js');
       $this->assertScriptContainsSrc('/oe_theme/js/ecl_auto_init.js');
 
       // Assert that the favicon provided by the theme is being used.
       $this->assertSession()->responseContains('/' . $active_theme . '/favicon.ico');
 
       // Assert that we do not load the EU component library by default.
-      $this->assertLinkNotContainsHref('/oe_theme/dist/eu/styles/ecl-eu-preset-website.css');
-      $this->assertLinkNotContainsHref('/oe_theme/dist/eu/styles/ecl-eu-preset-website-print.css');
+      $this->assertLinkNotContainsHref('/oe_theme/dist/eu/styles/ecl-eu.css');
+      $this->assertLinkNotContainsHref('/oe_theme/dist/eu/styles/ecl-eu-print.css');
       $this->assertLinkNotContainsHref('/oe_theme/css/style-eu.css');
 
-      $this->assertScriptNotContainsSrc('/oe_theme/dist/eu/scripts/ecl-eu-preset-website.js');
+      $this->assertScriptNotContainsSrc('https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.29.1/moment.min.js');
```

Comment:
Let's include this in our artifact, here is its project page: https://momentjs.com/ 
We can add it to our NPM dependencies and then simply copy it in our dist folder and include it as any other local file, as we do it already for svg4everybody.min.js:

![image](https://user-images.githubusercontent.com/153362/123594264-c825ad00-d7ef-11eb-90b8-a881eb0bd2ae.png)